android/java/org/chromium/chrome/browser/crypto_wallet/util/PendingTxHelper.java

```diff
@@ -112,4 +112,27 @@ public int compare(TransactionInfo lhs, TransactionInfo rhs) {
             runWhenDone.run();
         });
     }
+
+    public void removeTransactionFromPending(String accountName, String txId) {
+        HashMap<String, TransactionInfo[]> transactionInfos = getTransactions();
```

Comment:
why do we need `transactionInfos`? we can operate directly with `mTxInfos`.
Also I would recommend to change
`` to
`Collections.synchronizedMap(new HashMap<String, TransactionInfo[]>());`
as it looks it could be hit from diff threads right now

Reply:
yeah, it's fine to leave as HashMap if you don't need that method.